website/documentation/050-tutorials/content/howto/shell-to-node/_index.md

```diff
@@ -0,0 +1,387 @@
+The `AWSMachineClass` contains the machine image ([ami](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html)), machine-type, iam information, network-interfaces, subnets, security groups and attached volumes.
+
+Of course, the information can also be used to get the instance with the cloud provider CLI / API.
+
+## gardenctl ssh
+
+Using the node name of the problematic instance, we can use the `gardenctl ssh` command to get SSH access to the cloud provider 
+instance via an automatically set up [bastion host](https://en.wikipedia.org/wiki/Bastion_host).
+`gardenctl` takes care of spinning up the `bastion` instance, setting up the SSH keys, ports and security groups and opens a root shell on the target instance.
+After the SSH session ended, `gardenctl` deletes the created cloud provider resources.
+
+Use the following commands:
+
+First, target a Garden cluster containing all the Shoot definitions.
+```
+$ gardenctl target garden <target-garden>
+```
+
+Target an available Shoot by name. 
+This sets up the context, configures the `kubeconfig` file of the Shoot cluster and downloads the cloud provider credentials.
+Subsequent commands will execute in this context.
+
+```
+$ gardenctl target shoot <target-shoot>
+```
+
+This uses the cloud provider credentials to spin up the bastion and to open a shell on the target instance.
+```
+$ gardenctl ssh <target-node>
+```
+
+## SSH using a manually created Bastion on AWS
+
+In case you are not using `gardenctl` or want to control the bastion instance yourself, you can also manually set it up. 
+The steps described here are generally [the same as `gardenctl` uses internally](https://github.com/gardener/gardenctl/blob/10a537942b94234914758c0f6d053dc1cf218ecd/pkg/cmd/ssh_aws.go#L53-L52).
+The details of setting up the bastion host is different from cloud provider to cloud provider. 
+It can be generalized into the following steps:
+- Open port 22 on the target instance.
+- Create an instance / VM in a public subnet (bastion instance needs to have public ip address).  
+- Set-up security groups, roles and open port 22 for the bastion instance.
+
+The following diagram shows an overview how the SSH access to the target instance works:
+
+![520dc31009cf1d51e667551a5ae731e8.png](_resources/913441003e5641bc90249bdc07d55656.png)
+
+This guide demonstrates the setup of a bastion on AWS.
+
+**Prerequisites:**
+- The `Aws CLI` is set up.
+- Obtain target instance-id (see [here](#identifying-the-problematic-instance)).
+- Obtain the VPC ID the Shoot resources are created in. This can be found in the `Infrastructure` CRD in the `Shoot` namespace in the `Seed`.
+- Make sure that port 22 on the target instance is open (default for Gardener deployed instances).
+    - Extract security group via `$ aws ec2 describe-instances --instance-ids <instance-id>`
+    - Check for rule that allows inbound connections on port 22 via `$ aws ec2 describe-security-groups --group-ids=<security-group-id>`
```

Comment:
Prefer multi-line code snippets. When rendered in the documentation pages long code snippet single liners tend to overflow the page nastily. I haven't checked it locally, but please make sure it renders correctly or just use multi-line code  snippets to be on the safe site. Readers will also benefit from the one-click copy feature in this case. 

```suggestion
    - Check for rule that allows inbound connections on port 22 via 
      ```
      $ aws ec2 describe-security-groups --group-ids=<security-group-id>
      ```
```

See the other occurrences below too. I won't leave comments on each.